### DISCUSSION TITLE
Setting up multiple tunnes

### DISCUSSION THREAD

**Discussion by alexander-th** [homeassistant-apps/app-cloudflared#633] [Q&A] [answered]
Thanks for the great addon! I have used it so setup a local tunnel to my home assistant instance. 
I am also using home assistant to host the home assistant Jellyfin addon as a media server. Is it possible to use the cloudflared addon to setup a second tunnel to access Jellyfin? How could this be done?
Many Greetings,
Alex

**Comment by elcajon** [comment, marked as answer]
If I understand you correctly, are you trying to make another service available via the same tunnel in addition to Home Assistant, or is there a reason why you need a second tunnel for this? 
If you just want to make another service available via a domain, have a look at the add-on option for [additional_hosts](https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#option-additional_hosts).

**Comment by alexander-th** [comment]
Perfekt, Thanks a lot! This is what I was looking for. My apologies for
overlooking this in the documentation.

Max ***@***.***> schrieb am Do., 2. Mai 2024, 12:00:
